assets/js/blocks/catalog-sorting/block.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "woocommerce/catalog-sorting",
+	"version": "1.0.0",
+	"title": "Catalog Sorting",
+	"description": "Enable customers to change the sorting order of the products.",
+	"category": "woocommerce",
+	"keywords": [ "WooCommerce" ],
+	"supports": {
+		"align": true,
+		"color": {
+			"text": true
+		},
+		"typography": {
+			"fontSize": true,
+			"__experimentalFontFamily": true
+		}
```

Comment:
Color, font size, and font family aren't working both in the editor and on the front end because we apply the style to the wrapper `div`, not `select`.

Also, we shouldn't support `background` IMO. `background` and `text` color are enabled by default, so we need to explicitly disable background color.

<details>
<summary>Screenshot</summary>

<img width="724" alt="image" src="https://user-images.githubusercontent.com/5423135/212442824-f75adac1-5902-4f28-8d9f-906a9956bda0.png">

</details>

Reply:
Good catch, but can we actually add the text color class to the select? Since we are using the `woocommerce_catalog_ordering()` function, we don't have much control, should we disable text color as well? 🤔

Reply:
@albarin  We can use `inherit` for text color, font family, and font size of `select` to inherit style from the parent.

Reply:
👍